guides/server/uploads.md

```diff
@@ -82,35 +82,27 @@ Let's look at an annotated example:
 
 <%!-- use phx-drop-target with the upload ref to enable file drag and drop --%>
 <section phx-drop-target={@uploads.avatar.ref}>
-
-<%!-- render each avatar entry --%>
-<%= for entry <- @uploads.avatar.entries do %>
-  <article class="upload-entry">
-
+  <%!-- render each avatar entry --%>
+  <article :for={entry <- @uploads.avatar.entries} class="upload-entry">
     <figure>
       <.live_img_preview entry={entry} />
-      <figcaption><%= entry.client_name %></figcaption>
+      <figcaption>{entry.client_name}</figcaption>
     </figure>
 
     <%!-- entry.progress will update automatically for in-flight entries --%>
-    <progress value={entry.progress} max="100"> <%= entry.progress %>% </progress>
+    <progress value={entry.progress} max="100"> {entry.progress}% </progress>
 
     <%!-- a regular click event whose handler will invoke Phoenix.LiveView.cancel_upload/3 --%>
     <button type="button" phx-click="cancel-upload" phx-value-ref={entry.ref} aria-label="cancel">&times;</button>
 
     <%!-- Phoenix.Component.upload_errors/2 returns a list of error atoms --%>
-    <%= for err <- upload_errors(@uploads.avatar, entry) do %>
-      <p class="alert alert-danger"><%= error_to_string(err) %></p>
-    <% end %>
-
+    <p :for={err <- upload_errors(@uploads.avatar, entry)} class="alert alert-danger">{error_to_string(err)}</p>
   </article>
-<% end %>
-
-<%!-- Phoenix.Component.upload_errors/1 returns a list of error atoms --%>
-<%= for err <- upload_errors(@uploads.avatar) do %>
-  <p class="alert alert-danger"><%= error_to_string(err) %></p>
-<% end %>
 
+  <%!-- Phoenix.Component.upload_errors/1 returns a list of error atoms --%>
```

Comment:
if the EEx syntax is going away it seem odd to keep it around just for comment tags. Why not a `{% %}` style?

Reply:
It is not going away though, it is not possible to do `{if ... do}`, `{for ... do}`, etc. (Not that people should reach for those given :if and :for exists.) We can't use `{...}` interpolation inside `<script>` and `<style>` either and for that `<%= ... %>` is the way to go.

Reply:
The `<%!-- --%>` comment syntax was inspired by Surface's `{!-- --}`. If we're moving to curly braces in the HTML, I'd vote we use the curly comments too.

Reply:
+1 for `{!-- --}` style